### ISSUE TITLE
l

### ISSUE THREAD

**Issue by charmJohn** [XLR-man/XLR-man.github.io#1]
Hello, I'm encountering some difficulties in configuring the environment for llnerf. Since you've successfully implemented llnerf, I'd like to seek your assistance.

**Comment by XLR-man** [comment]
Yeah, we can talk to each other, because I have some questions, too
